# Preprocessing
This notebook splits the dataset into train and test data and saves it in the data - processed folder.

In [27]:
#import packages
import pandas as pd
from sklearn.model_selection import train_test_split

In [28]:
#load dataset
df = pd.read_csv('../data/raw/heart.csv')

In [29]:
#split train and test data
train_df, test_df = train_test_split(df, test_size=.2, random_state=42)
train_df.shape, test_df.shape

((820, 14), (205, 14))

In [30]:
#save train and test data in processed folder
train_df.to_csv('../data/processed/train_data.csv')
test_df.to_csv('../data/processed/test_data.csv')

# Key Takeaways
- training data is 820 records
- test data is 205 records